app/models/ticket_holder.py

```diff
@@ -16,7 +16,7 @@ class TicketHolder(db.Model):
 
     id = db.Column(db.Integer, primary_key=True)
     firstname = db.Column(db.String, nullable=False)
-    lastname = db.Column(db.String)
+    lastname = db.Column(db.String, nullable=False)
```

Comment:
@mayank8318 revert this change and remove migration file as well.

Reply:
@poush Ok ☺ I will make the changes by Monday once my exams get over. Is that okay?